src/containers/DemoContainer.tsx

```diff
@@ -30,6 +31,24 @@ const DemoContainer: React.FC = () => {
   // conditional rendering of text when "Clear Cache" button is clicked
   const [cacheIsClear, setCacheIsClear] = React.useState<Boolean>(false);
 
+  // conditional rendering of DenoStore demonstration paragraph when info icon is clicked
+  const [demoParagraph, setDemoParagraph] = React.useState<Boolean>(false);
+
+  // state of the button based on whether Demo Explanation is visible or hidden
+  const [demoExplanationClicked, setDemoExplanationClicked] =
```

Comment:
Can we use just one piece of state here? demoExplanationOpen as demoExplanationClicked

Reply:
Resolved.